### PR TITLE
Streamline APM Product Overview

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do? What is the motivation?

Streamline APM product overview so it's easier for users to on board.

[Preview](https://docs-staging.datadoghq.com/brett0000FF/apm-product-overview/tracing/)

To do 
- [x] Validate use cases
- [ ] Clean up links

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/_index.md [hunk 1]
```diff
@@ -52,70 +40,60 @@ cascade:
...
+## Use cases
 
-[Generate metrics][8] with 15-month retention from all ingested spans to create and monitor key business and performance indicators over time.
+Discover some ways Datadog APM can help support your use cases:
 
-{{< img src="tracing/index/SpantoMetricsPreview.png" alt="Generate Custom Metrics from ingested spans" style="width:100%;">}}
+| You want to...| How Datadog APM can help |
+| ----------- | ----------- |
+| Understand how requests flow through your system. | [Trace Explorer][21] visualizes and analyzes end-to-end traces across distributed services. |
+| Monitor service health and performance of individual services. | The [service page][26] helps you assess service health by analyzing performance metrics, tracking deployments, identifying problematic resources. |
```
Comment (dussault-antoine) [inline content/en/tracing/_index.md:62]:
```suggestion
| Monitor service health and performance of individual services. | The [service][26] and [resource pages][28] helps you assess service health by analyzing performance metrics, tracking deployments, identifying problematic resources. |
```

File: content/en/tracing/_index.md [hunk 2]
```diff
@@ -137,3 +115,12 @@ Traces start in your instrumented applications and flow into Datadog. For high-t
...
 [18]: /tracing/trace_collection/custom_instrumentation/
+[19]: /tracing/metrics/
+[20]: /tracing/other_telemetry/
+[21]: /tracing/trace_explorer/
+[22]: /tracing/trace_collection/automatic_instrumentation/single-step-apm/
+[23]: /agent/
+[24]: /tracing/metrics/metrics_namespace/
+[25]: /tracing/metrics/runtime_metrics/
+[26]: /tracing/services/service_page/
+[27]: /tracing/trace_collection/single-step-apm/
```
Comment (dussault-antoine) [inline content/en/tracing/_index.md:126]:
```suggestion
[27]: /tracing/trace_collection/single-step-apm/
[28]: /tracing/services/resource_page/
```

File: content/en/tracing/_index.md [hunk 3]
```diff
@@ -52,70 +40,60 @@ cascade:
...
-## Generate custom metrics from spans
+## Use cases
 
-[Generate metrics][8] with 15-month retention from all ingested spans to create and monitor key business and performance indicators over time.
+Discover some ways Datadog APM can help support your use cases:
 
-{{< img src="tracing/index/SpantoMetricsPreview.png" alt="Generate Custom Metrics from ingested spans" style="width:100%;">}}
+| You want to...| How Datadog APM can help |
+| ----------- | ----------- |
+| Understand how requests flow through your system. | [Trace Explorer][21] visualizes and analyzes end-to-end traces across distributed services. |
```
Comment (dussault-antoine) [inline content/en/tracing/_index.md:61]:
```suggestion
| Understand how requests flow through your system. | Use the [Trace Explorer][21] to query and visualize end-to-end traces across distributed services. |
```

File: content/en/tracing/_index.md [hunk 4]
```diff
@@ -52,70 +40,60 @@ cascade:
...
+| Monitor service health and performance of individual services. | The [service page][26] helps you assess service health by analyzing performance metrics, tracking deployments, identifying problematic resources. |
+| Correlate traces with DBM, RUM, logs, synthetics, and profiles. | [Correlate APM Data with Other Telemetry][20] to give more context to your data for more comprehensive analysis. |
+| Control how data flows into Datadog. | [Ingestion Controls][6] adjust service-level ingestion configuration and sampling rates. [Retention filters][7] controls which spans are retained for 15 days. |
```
Comment (dussault-antoine) [inline content/en/tracing/_index.md:64]:
```suggestion
| Control how data flows into Datadog. | Use [Ingestion Controls][6] to adjust ingestion configuration and sampling rates by service and resource. Use [Retention filters][7] to choose which spans to retain for 15 days. |
```

File: content/en/tracing/_index.md [hunk 5]
```diff
@@ -52,70 +40,60 @@ cascade:
...
+## Use cases
 
-[Generate metrics][8] with 15-month retention from all ingested spans to create and monitor key business and performance indicators over time.
+Discover some ways Datadog APM can help support your use cases:
 
-{{< img src="tracing/index/SpantoMetricsPreview.png" alt="Generate Custom Metrics from ingested spans" style="width:100%;">}}
+| You want to...| How Datadog APM can help |
+| ----------- | ----------- |
+| Understand how requests flow through your system. | Use the [Trace Explorer][21] to query and visualize end-to-end traces across distributed services. |
+| Monitor service health and performance of individual services. | The [service][26] and [resource pages][28] helps you assess service health by analyzing performance metrics, tracking deployments, identifying problematic resources. |
```
Comment (buraizu) [inline content/en/tracing/_index.md:62]:
```suggestion
| Monitor service health and performance of individual services. | The [service][26] and [resource pages][28] help you assess service health by analyzing performance metrics, tracking deployments, and identifying problematic resources. |
```

File: content/en/tracing/_index.md [hunk 6]
```diff
@@ -52,70 +40,60 @@ cascade:
...
+| Correlate traces with DBM, RUM, logs, synthetics, and profiles. | [Correlate APM Data with Other Telemetry][20] to give more context to your data for more comprehensive analysis. |
```
Comment (buraizu) [inline content/en/tracing/_index.md:63]:
```suggestion
| Correlate traces with DBM, RUM, logs, synthetics, and profiles. | [Correlate APM Data with Other Telemetry][20] to give context to your data for more comprehensive analysis. |
```

**Review by buraizu — APPROVED** [review]
Looks good overall, approving with some minor feedback.

**Pull request merged**